templates/travis/.travis/before_install.sh.j2

```diff
@@ -24,10 +27,12 @@ then
 {%- for repository in ["pulpcore", "pulp-smash", "pulp-openapi-generator"] + additional_plugins | map(attribute="name") | list %}
   export {{ repository | upper | replace("-", "_") }}_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/{{ repository }}\/pull\/(\d+)' | awk -F'/' '{print $7}')
 {%- endfor %}
+  echo $COMMIT_MSG | sed -n -e 's/.*CI Base Image:\s*\([-_/[:alnum:]]*:[-_[:alnum:]]*\).*/ci_base: "\1"/p' >> .travis/vars/main.yaml
```

Comment:
It would be cool to see it in the commit message of this PR

Reply:
i have an example here: https://github.com/pulp/pulp_container/pull/116

Reply:
cool!